apps/docker-base/Makefile.docker-base

```diff
@@ -1,35 +1,40 @@
-# -*- makefile -*-
```

Comment:
Please don't get rid of this line. It's magic that helps some editors.

Reply:
Thanks for your reply.
  I will check all  this